crates/apub/src/activities/community/add_mod.rs

```diff
@@ -32,6 +32,7 @@ use lemmy_utils::LemmyError;
 use lemmy_websocket::LemmyContext;
 
 impl AddMod {
+  #[tracing::instrument(skip(community, added_mod, actor, context))]
```

Comment:
So effectively these all include only the request_counter? Dont think thats useful, it would only be interesting if the limit is reached, and in that case its logged anyway. So i would probably just use skip_all everywhere.